deployment/content-localization-on-aws-auth.yaml

```diff
@@ -274,6 +277,17 @@ Resources:
                     ],
                     "Effect": "Allow",
                     "Resource": "${searchdomain}/*"
+                  },
+                  {
```

Comment:
Does this new policy also apply to authorized but non-admin users?
Meaning, would authorized non-admin users also be able to upload without the kms issue?

Reply:
The solution only creates 1 user group, and it's associated to the Admin group/role. In the IG, the directions for creating a user is to create them under this 1 group that gets created when deployed. To create more user groups after deployment, you'd have to copy/paste the permissions you want for this separate group.